src/ktn_meta_SUITE.erl

```diff
@@ -57,7 +57,7 @@ dialyzer(Config) ->
   Dirs = dirs(Config),
   Warnings =
     case test_server:lookup_config(dialyzer_warnings, Config) of
-      undefined -> [error_handling, race_conditions, unmatched_returns];
+      undefined -> [error_handling, race_conditions, unmatched_returns, unknown];
```

Comment:
Line 60 is too long:       undefined -> [error_handling, race_conditions, unmatched_returns, unknown];.